presto-main/src/main/java/com/facebook/presto/metadata/SqlScalarFunctionBuilder.java

```diff
@@ -99,6 +100,24 @@ public SqlScalarFunctionBuilder nullableArguments(List<Boolean> nullableArgument
         return this;
     }
 
+    public SqlScalarFunctionBuilder nullFlags(boolean... nullFlags)
+    {
+        requireNonNull(nullFlags, "nullableArguments is null");
+
+        ImmutableList.Builder<Boolean> nullFlagsBuilder = ImmutableList.builder();
+        for (boolean flag : nullFlags) {
+            nullFlagsBuilder.add(flag);
+        }
+        this.nullFlags = nullFlagsBuilder.build();
+        return this;
+    }
+
+    public SqlScalarFunctionBuilder nullFlags(List<Boolean> nullFlags)
+    {
+        this.nullFlags = copyOf(requireNonNull(nullFlags, "nullableArguments is null"));
```

Comment:
I agree. As you noticed, that is a copy paste. I'd like to make cumulative fix  in a separate commit.